### DISCUSSION TITLE
[SOLVED] AdBlock needs to be configured

### DISCUSSION THREAD

**Discussion by PrzemekRyk** [martinrotter/rssguard#704] [Q&A]
Hi all,

Long story short - two days ago everything worked fine. I could use RSS Guard with AdBlock enabled without any problems. I've turned off Guard, I'm starting it today - and this damn message appears: **AdBlock needs to be configured**. What the heck? :( OK - it happens to me once, so I've remembered, that node.js should be installed. I've checked - it is. Version 17.6.0.

OK. New version is available (18.0.0). Downloaded, installed. I'm starting RSS Guard, enable AdBlock. Seems wor… no. The same happens. I've rolled back to older version than 17.6.0 (16.14.2) - the same result.

Any ideas what should I do, to have AdBlock working in RSS Guard? I've tried to report it as a bug, but it was closed and rejected as impossible to repeat. OK - maybe it's something wrong at my PC, but also maybe someone of you have solved this problem?

Versions:

- RSS Guard: 4.2.1
- node.js tested: 16.14.2 / 17.6.0 / 18.0.0
- Windows 8.1 x64

**Comment by martinrotter** [comment]
Why (the hell, sorry) do you open another ticket/discussion? Go back to

https://github.com/martinrotter/rssguard/issues/702
